search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -242,11 +245,24 @@
     {:name (or archive-center umm-spec-util/not-provided)
      :subOrganizationOf hierarchy}))
 
+(defn check_for_modis_aster
+  "Logic to test for collections that use the MODIS or ASTER instruments as
+  these are the only ones that should be included in the data.gov output as
+  defined by The Data Curation for Discovery (DCD) team at Marshall Space Flight
+  Center (MSFC)"
+  [x]
+  (let [short-name (:short-name x)]
```

Comment:
Prefer destructured syntax and more verbose input value than x
'let [{:keys [short-name]} collection]`